cephalopod/webinterface/misc.py

```diff
@@ -13,10 +13,19 @@
 from . import bp
 
 
-def get_extra_fields(server_list):
+def get_all_server_fields(server_list):
     extra_fields = set()
     for server in server_list:
-        extra_fields.update(server.crawled_data or {})
+        extra_fields.update(get_server_fields(server) or {})
+    return sorted(extra_fields)
+
+
+def get_server_fields(server):
+    extra_fields = set()
+    fields_settings = current_app.config['CRAWLED_FIELDS_SETTINGS']
+    if server.crawled_data:
+        data = {k: v for k, v in server.crawled_data.iteritems() if fields_settings.get(k, False) is not None}
```

Comment:
Do None/False have different meanings? If False as no special purpose I'd consider anything falsy as "ignored" like you actually already do in the jinja loop above